scheduling-manager/src/main/java/com/amazonaws/blox/scheduling/manager/ManagerApplication.java

```diff
@@ -14,33 +14,34 @@
  */
 package com.amazonaws.blox.scheduling.manager;
 
-import com.amazonaws.blox.lambda.LambdaFunction;
 import com.amazonaws.blox.lambda.AwsSdkV2LambdaFunction;
+import com.amazonaws.blox.lambda.LambdaFunction;
 import com.amazonaws.blox.scheduling.SchedulingApplication;
 import com.amazonaws.blox.scheduling.scheduler.SchedulerInput;
 import com.amazonaws.blox.scheduling.scheduler.SchedulerOutput;
 import com.amazonaws.blox.scheduling.state.ECSStateClient;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import software.amazon.awssdk.services.lambda.LambdaAsyncClient;
 
 @Configuration
-@ComponentScan("com.amazonaws.blox.scheduling.manager")
+@ComponentScan({
+  "com.amazonaws.blox.scheduling.manager",
+  "com.amazonaws.blox.scheduling.state",
+})
 public class ManagerApplication extends SchedulingApplication {
```

Comment:
would it be possible to put all the configuration in one package so it's easy to find?

Reply:
In this case, it's not the configuration that's in a different package, it's the `@Component` annotation on the state stuff.

Reply:
What do you mean? I think the schedulingapplication configuration is not in com...manager

Reply:
Oh, wait, dammit github highlighting. Do you mean "why is SchedulingApplication in a different package"? If so, the reason is that it's the shared config between all components of the scheduling-manager package. If I were to put it in any of the subpackages, then it would no longer be clear that it's common to all of them.

An alternative could be to add a separate top-level `com.amazonaws.blox.scheduling.config` package, and add all the config in there. Is that what you were alluding to?